src/natcap/invest/urban_nature_access.py

```diff
@@ -463,8 +554,11 @@ def _admin_level_supply_demand(
         avg_greenspace_supply_demand = (
             greenspace_stats[feature_id]['sum'] /
             population_stats[feature_id]['sum'])
-
         feature.SetField(supply_sum_fieldname, avg_greenspace_supply_demand)
+        feature.SetField(
+            undersupply_fieldname, undersupplied_stats[feature_id]['sum'])
+        feature.SetField(
+            oversupply_fieldname, oversupplied_stats[feature_id]['sum'])
```

Comment:
If we add anymore stats to the vector aggregate we might want to move this into an iteration. But, I think it's fine at 3.